src/renderer/components/+user-management/+role-bindings/dialog.tsx

```diff
@@ -91,17 +91,15 @@ export class RoleBindingDialog extends React.Component<Props> {
     const serviceAccounts = Array.from(this.selectedAccounts, sa => ({
       name: sa.getName(),
       kind: "ServiceAccount" as const,
-      namespace: this.bindingNamespace,
+      namespace: sa.getNs(),
     }));
     const users = Array.from(this.selectedUsers, user => ({
       name: user,
-      kind: "User" as const,
-      namespace: this.bindingNamespace,
```

Comment:
This is not what we want? Or am I missunderstanding what the namespace field in a subject binding is for?

Reply:
Ah so it is only for service accounts. Thanks.